KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java

```diff
@@ -250,11 +328,15 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         subInstructionLbl.setFont(new java.awt.Font("Times New Roman", 0, 14)); // NOI18N
         subInstructionLbl.setText("Select a type, click build, and then click where it shoud go.");
 
-        diceRollLbl.setFont(new java.awt.Font("Times New Roman", 0, 24)); // NOI18N
-        diceRollLbl.setText(" ");
+        backNoSaveBtn.setText("< Exit without saving");
+        backNoSaveBtn.addActionListener(new java.awt.event.ActionListener() {
+            public void actionPerformed(java.awt.event.ActionEvent evt) {
+                backNoSaveBtnActionPerformed(evt);
+            }
+        });
 
-        diceRollPromptLbl1.setFont(new java.awt.Font("Times New Roman", 0, 18)); // NOI18N
-        diceRollPromptLbl1.setText("You rolled a: ");
+        titleLbl.setFont(new java.awt.Font("Times New Roman", 1, 40)); // NOI18N
+        titleLbl.setText("Settlers of Catan");
```

Comment:
GH is thinking I changed these lines. They are in fact NOT related and for 2 completely different Swing variables.